sync2/poller.go

```diff
@@ -550,7 +570,7 @@ func (p *poller) poll(ctx context.Context, s *pollLoopState) error {
 		}
 	}
 	if s.since == "" {
-		p.logger.Info().Msg("Poller: valid initial sync response received")
+		p.logger.Info().Str("user_id", p.userID).Str("device_id", p.deviceID).Msg("Poller: valid initial sync response received")
```

Comment:
If the poller is created via here, yes:

https://github.com/matrix-org/sliding-sync/blob/c215503854b4362f5d3d2b1f2b8756d9d5e5c3ef/sync2/handler2/handler.go#L158

and also via here, yes:

https://github.com/matrix-org/sliding-sync/blob/c215503854b4362f5d3d2b1f2b8756d9d5e5c3ef/sync2/handler2/handler.go#L556

Odd. I could have sworn I didn't see the IDs in my test. Will double check

Reply:
Oh, it'll be because some of the tests pass in a "bare" logger: https://github.com/matrix-org/sliding-sync/blob/d659824edf06006491e2ddae89d6210f909718f5/sync2/poller_test.go#L57